kin-devplatform-sdk/src/main/java/kin/devplatform/KinEcosystemInitiator.java

```diff
@@ -209,7 +194,8 @@ public void onMigrationStart() {
 			public void onReady(IKinClient kinClient) {
 				Logger.log(new Log().priority(Log.DEBUG).withTag(TAG).text("onReady"));
 				try {
-					handleKinClientReady(kinClient, context, appId, signInData, true, loginCallback);
+					handleKinClientReady(kinClient, migrationManager, context, appId, signInData, true, loginCallback);
+					// TODO: 07/02/2019 don't remember why we added the didMigrationStarted but if not started then i don't see a reason why not calling onFinish
```

Comment:
onReady will be called even if no migration was done, we need a way to tell the developer about migration only when it really happened, i.e. when `onMigrationStarted` was triggered

Reply:
Thats a problem because onMigrationStarted will be called also if the backup account was never on the device.
This means that even if the account will not be migrated at the end the method will still be called in that case